src/content/changes/kubectl-gs/kubectl-gs/v2.55.0.md

```diff
@@ -0,0 +1,16 @@
+---
+# Generated by scripts/aggregate-changelogs. WARNING: Manual edits to this files will be overwritten.
+changes_categories:
+- kubectl gs
+changes_entry:
+  repository: giantswarm/kubectl-gs
+  url: https://github.com/giantswarm/kubectl-gs/blob/master/CHANGELOG.md#2550---2024-05-14
+  version: 2.55.0
+  version_tag: v2.55.0
+date: '2024-05-14T12:43:17'
+description: Changelog entry for giantswarm/kubectl-gs version 2.55.0, published on
+  14 May 2024, 12:43.
+title: kubectl-gs release v2.55.0
+---
+
+
```

Comment:
This is invalid (to be fixed via https://github.com/giantswarm/kubectl-gs/pull/1328)